e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -34,7 +34,8 @@
     private static final String EHR_CONTINUE = "ehrContinue";
     private static final String GPC_STRUCTURED_FILENAME_EXTENSION = "_gpc_structured.json";
     private static final String DOCUMENT_ID = "07a6483f-732b-461e-86b6-edb665c45510";
-    
+    public static final String ACCEPTED_ACKNOWLEDGEMENT_TYPE_CODE = "AA";
+
     @Test
     public void When_ExtractRequestReceived_Expect_ExtractStatusAndDocumentDataAddedToDatabase() throws Exception {
         String conversationId = UUID.randomUUID().toString();
```

Comment:
Although the UUID logic hasn't been changed in this PR.
Could we use the randomIdGeneratorService for the tests in this class?